packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,180 @@
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta Event Hook calls and responses.
+* Implement Okta Event Hooks using a local application and the Hookdeck utility.
+* Preview and test an Okta Event Hook and review the call details with Hookdeck.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/)
+* A local application. Or see [Create a local application](#create-a-local-application) in this guide to install the sample code application.
+* The Hookdeck utility. See [Install Hookdeck](#install-hookdeck).
+
+**Sample code**
+
+* See [nodejs-webhook-server-example](https://github.com/hookdeck/nodejs-webhook-server-example) for this project's sample code. Installation instructions are available in the next section, [Create a local application](#create-a-local-application).
+
+---
+
+## Create a local application
+
+<StackSelector snippet="sample-app" noSelector/>
+
+## Install Hookdeck
+
+If you already have Hookdeck installed, move on to [Run Hookdeck](#run-hookdeck). If not, follow the installation instructions at [Using The Hookdeck CLI](https://hookdeck.com/docs/using-the-cli) to install in your sample application folder (or the location of your choice).
+
+Some installation notes:
+
+* You don't need to have an account to install and run Hookdeck, but creating a free account provides more features as well as API authentication.
+* The installation uses the [Homebrew](https://brew.sh/) package manager.
+
+### Run Hookdeck
+
+After installing Hookdeck, ensure that it's running by creating a "tunnel" into a local port (1337 in this example). Run the following command in your terminal:
+
+```terminal
+nodejs-webhook-server-example> hookdeck listen 1337
+```
+
+Complete the interactive session that follows to define the information about the Hookdeck session:
+
+* **What source should you select?**
+
+    Enter: `Create new source`
+
+* **What should your new source label be?**
+
+    Enter: `okta`
+
+* **What path should the webhooks be forwarded to (i.e.: /webhooks)?**
+
+    Enter: `/okta-webhooks-endpoint`
+
+* **What's the connection label (i.e.: My API)?**
+
+    Enter: `My Okta Response Server`
+
+With this data, Hookdeck creates a session and URL to use for sending requests. If you see the following content in your terminal, Hookdeck is running successfully:
+
+![A screen shot of a terminal that displays a Hookdeck session status. The session status contains urls that tunnel into the local port.](/img/hookdeck-and-event-hooks-session.png)
+
+> **Note:** Copy the forwarding **Webhook URL** that is available from the Hookdeck terminal session. For example, `https://events.hookdeck.com/e/src_s8lCGfojGBPj8L0lszAZl6fD`. You'll use this URL when setting up your Okta Event Hook.
```

Comment:
Yes, we go over this in the Event Hook set up procedure. The PM wanted to highlight this URL,  so, users would know what URL to use later on. But instead of making it a task, I'll remove "Copy", and make it more of a "note", i.e.: 

**Note:** You'll use the **Webhook URL** that is available from the Hookdeck terminal session when setting up your Okta Event Hook.